frontends/qt/server/Makefile.macosx

```diff
@@ -8,10 +8,11 @@ GOOS=darwin
 darwin:
 	CGO_CFLAGS="-g ${GODARWINSECFLAGS} ${CFLAGS}" \
 	CGO_LDFLAGS="${GODARWINLDFLAGS} ${LFLAGS}" \
+ 	MACOSX_DEPLOYMENT_TARGET=${MACOS_MIN_VERSION} \
```

Comment:
Turns out, there's one space and one tab character. @jstrnbrg teach your editor to use appropriate indentation. :)

![image](https://user-images.githubusercontent.com/25405/80806969-466ecb00-8bbc-11ea-9052-119a7980258e.png)

Reply:
should be fixed now:)